### PR TITLE
Add docs and logo

### PR DESCRIPTION
**Pull request by jkisk**

### CODE REVIEW

File: docs/_index.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+import (
+	"fmt"
+	slack "github.com/pulumi/pulumi-slack/sdk/go/slack"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+
+		channel, err := slack.Conversation(ctx, "test", &slack.ConversationArgs{
```
Comment (stack72) [inline docs/_index.md:54]: gofmt isn't quite correct here as a nit

File: docs/_index.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+```csharp
+using Pulumi;
+using Pulumi.Slack;
+
+class SlackChannel : Channel
+{
+    public SlackChannel()
+    {
+        var channel = new Conversation("test", new ConversationArgs{
+            topic: "my topic",
```
Comment (stack72) [inline docs/_index.md:81]: yeah these are not correct:

Topic = "my topic",
IsPrivate = False,

**Review by stack72 — CHANGES_REQUESTED** [review]
Small not regarding formatting of the examples for Go and DotNet

File: docs/_index.md (hunk 2 above)
Comment (jkisk) [reply docs/_index.md:81]: thank you!

**Pull request merged**